qemu/tests/block_stream_check_backingfile.py

```diff
@@ -36,6 +37,13 @@ def check_backingfile(self):
                    "no backing file")
             raise error.TestFail(msg)
 
+    def check_backingfile_exist(self):
+        backingfile = self.get_backingfile()
+        if backingfile != self.base_image:
+            msg = "The backing file from monitor does not meet expectation. "
+            msg += "It should be %s, now is %s." % (self.base_image, backingfile)
+            raise error.TestFail(msg)
```

Comment:
As https://github.com/avocado-framework/avocado-vt/pull/941#issuecomment-288009679 mentioned, better to use `test` instance to report the status.

http://avocado-framework.readthedocs.io/en/latest/api/test/avocado.html